contracts/AnchorVault.vy

```diff
@@ -229,17 +142,15 @@ def petrify_impl():
 def emergency_stop():
```

Comment:
maybe just 'stop'

Reply:
`stop` is a reserved word in vyper, so lets use the `pause`